packages/go-musicfox/build.sh

```diff
@@ -0,0 +1,38 @@
+TERMUX_PKG_HOMEPAGE=https://github.com/go-musicfox/go-musicfox
+TERMUX_PKG_DESCRIPTION="A netease music player in terminal."
+TERMUX_PKG_LICENSE="MIT"
+TERMUX_PKG_MAINTAINER="@anhoder"
+TERMUX_PKG_VERSION="4.4.0"
+TERMUX_PKG_DEPENDS="libc++, libflac, libvorbis"
+TERMUX_PKG_SRCURL=git+https://github.com/go-musicfox/go-musicfox
+TERMUX_PKG_GIT_BRANCH=master
+TERMUX_PKG_AUTO_UPDATE=true
+TERMUX_PKG_UPDATE_TAG_TYPE="newest-tag"
+TERMUX_PKG_UPDATE_VERSION_REGEXP="\d+\.\d+\.\d+"
+
+termux_step_post_get_source() {
```

Comment:
`termux_step_post_get_source()` won't need to be overwritten when building from a release tarball